_posts/2018-04-09-themeswitcher.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: ThemeSwitcher
+date: 2018-04-09
+categories:
+    - projects
+description: A webservice that allows a user to select a bootstrap theme and an api to serve that theme to other sites that desire to implement themeswitcher.
```

Comment:
could you add an image attribute w/ a screenshot of the project? Should be in the form:
`image: https://urlgoes.here/`